app/controllers/invitations_controller.rb

```diff
@@ -0,0 +1,83 @@
+class InvitationsController < ApplicationController
+  before_action :authenticate_with_pre_login_destination, only: [:show]
+  before_action :set_organization, only: [:create]
+
+  def create
```

Comment:
@johndbritton after this refactor, I think `#create` is much better. However I'm still looking at a way to better handle the creation or finding of teams.

Suggestions?